plugins/inputs/jenkins/jenkins_test.go

```diff
@@ -0,0 +1,92 @@
+package jenkins
```

Comment:
Generally speaking, we need more tests here.  I think we should probably mock go-jenkins, but alternatively we could use httptest.

Reply:
👍 
created a mocking jenkins server,
added a lot of tests, now 78.5% coverage, for newInstance, gather jobs, sub jobs, gather nodes.